automation/DeployToHPVS.bashlib

```diff
@@ -489,15 +489,15 @@ config_file_wizard(){
     # Config variable: APIKey                                                  #
     #--------------------------------------------------------------------------#
     PROMPT_USER_READARGS="-s"
-    prompt_user "Enter an IAM key (platform API key) for your IBM Cloud account" false
+    prompt_user "Enter an IAM key (platform API key) for your IBM Cloud account. An example API key looks like this k-y2lsNxfeKbaidcCG6kQVypLDR_WjmDPe-2Uuq_PTYx" false
     unset PROMPT_USER_READARGS
```

Comment:
new line....

and add this

"Login to your IBM Cloud account and navigate to Manage > Access (IAM) > API keys for more information."